onnxruntime/test/python/onnxruntime_test_ort_trainer.py

```diff
@@ -106,7 +106,7 @@ def runBertTrainingTest(gradient_accumulation_steps, use_mixed_precision, allred
 
         learning_rate = torch.tensor([lr]).to(device)
         if use_mixed_precision:
-            loss_scale = torch.tensor(loss_scaler.loss_scale_).to(device)
+            loss_scale = torch.tensor([loss_scaler.loss_scale_]).to(device)
```

Comment:
We are changing loss_scale from scalar(shape []) to 1-d tensor(shape [1]), is this change expected?

Reply:
Yep, it's consistent with the learning rate input.